Prac_06/guitars.py

```diff
@@ -0,0 +1,40 @@
+"""
+Began 3:20
+
+"""
+from Prac_06.guitar import Guitar
+
+"""Create a program that reads guitar inputs and sorts them to a list"""
+
+
+def main():
+    guitars_list = []
```

Comment:
I believe your list naming is incorrect, 

For example, a list of guitars should be named - "guitars" not guitars_list. Refer to list naming conventions but I am almost certain about this